demo/assets/js/app.js

```diff
@@ -27,84 +21,18 @@ window.addEventListener('phx:page-loading-start', _info => topbar.show(250))
 window.addEventListener('phx:page-loading-stop', _info => topbar.hide())
 
 /**
- * theme selector
+ * Theme Selector
  */
-
-const Hooks = {}
-
-// We want this to run as soon as possible to minimize
-// flashes with the old theme in some situations
-const storedTheme = window.localStorage.getItem('backpexTheme')
-if (storedTheme != null) {
-  document.documentElement.setAttribute('data-theme', storedTheme)
-}
-Hooks.BackpexThemeSelector = {
-  mounted () {
-    const form = document.querySelector('#backpex-theme-selector-form')
-    const storedTheme = window.localStorage.getItem('backpexTheme')
-
-    // Marking current theme as active
-    if (storedTheme != null) {
-      const activeThemeRadio = form.querySelector(
-        `input[name='theme-selector'][value='${storedTheme}']`
-      )
-      activeThemeRadio.checked = true
-    }
-
-    // Event listener that handles the theme changes and store
-    // the selected theme in the session and also in localStorage
-    window.addEventListener('backpex:theme-change', async (event) => {
-      const cookiePath = form.dataset.cookiePath
-      const selectedTheme = form.querySelector(
-        'input[name="theme-selector"]:checked'
-      )
-      if (selectedTheme) {
-        window.localStorage.setItem('backpexTheme', selectedTheme.value)
-        document.documentElement.setAttribute(
-          'data-theme',
-          selectedTheme.value
-        )
-        await fetch(cookiePath, {
-          body: `select_theme=${selectedTheme.value}`,
-          method: 'POST',
-          headers: {
-            'Content-type': 'application/x-www-form-urlencoded',
-            'x-csrf-token': csrfToken
-          }
-        })
-      }
-    })
-  }
-}
+BackpexHooks.BackpexThemeSelector.setStoredTheme()
 
 /**
  * phoenix_live_view
  */
 const csrfToken = document.querySelector("meta[name='csrf-token']").getAttribute('content')
 
 const liveSocket = new LiveSocket('/live', Socket, {
-  dom: {
-    onBeforeElUpdated (from, to) {
-      if (from._x_dataStack) {
-        window.Alpine.clone(from, to)
-      }
-    },
-    onNodeAdded (node) {
-      // Mimic autofocus for dynamically inserted elements
-      if (node.nodeType === window.Node.ELEMENT_NODE && node.hasAttribute('autofocus')) {
-        node.focus()
-
-        if (node.setSelectionRange && node.value) {
-          const lastIndex = node.value.length
-          node.setSelectionRange(lastIndex, lastIndex)
-        }
-      }
```

Comment:
I can't remember why we added this, but I wanted to add a comment in case this wasn't Alpine's reason (I don't see any Alpine specific code like in `onBeforeElUpdated`).